apps/service_providers/tests/test_prompt_context.py

```diff
@@ -16,8 +16,9 @@ def mock_session():
     session.participant.user = None
 
     proxy_mock = Mock()
-    proxy_mock.get_data.return_value = "participant data"
+    proxy_mock.get.return_value = "participant data"
```

Comment:
Should the return value not be a `dict`?

Reply:
+1

Reply:
nice thank you